files/master/buster_syncd.patch

```diff
@@ -0,0 +1,174 @@
+diff --git a/platform/marvell-arm64/docker-saiserver-mrvl.mk b/platform/marvell-arm64/docker-saiserver-mrvl.mk
```

Comment:
There is already syncd_buster.patch in "files/master" folder.

Reply:
Rajkumar, as discussed will add it in 202106 specific folder. As noted see some differences in the patch which is already there for master.